src/app/components/Account/index.tsx

```diff
@@ -119,10 +119,7 @@ export const Account: FC<AccountProps> = ({ account, token, isLoading, tokenPric
             <>
```

Comment:
This uncovered a bug in account loading state:  
- open https://fb893d61.oasis-explorer.pages.dev/mainnet/emerald/token
- click yuzu's contract address 0xf02b3e437304892105992512539F769423a515Cb
- token name appears with a delay

![delay](https://github.com/oasisprotocol/explorer/assets/3758846/11a8882b-1bcd-4307-8462-a0eee8c74e12)

Reply:
Fixed